### PR TITLE
added baselines and ignoring serac output

### PR DESCRIPTION
**Pull request by chapman39**

### CODE REVIEW

File: integration/test.ats [hunk 1]
```diff
@@ -78,36 +76,46 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
...
+            clas=checker_args,
+            label=name,
+            nosrun=True)
+    else:
+        # Rebaseline
+        if os.path.exists(test_summary_file):
+            print("Rebaselining {0}".format(name))
+            shutil.copyfile(test_summary_file, baseline_summary_file)
+        else:
+            print("Warning: tried to copy test file ({0}) that doesn't exist".format(
```
Comment (white238) [inline integration/test.ats]: avoid putting paths in the middle of a warning message. it makes it hard to read.  do something like this:

```
Warning: Something has occurred with file:
  <path>
```

File: integration/test.ats [hunk 2]
```diff
@@ -78,36 +76,46 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
...
 define(tolerance_test=tolerance_test)
 
-
 #
 # Add individual tests found in subdirectories
 #
 
-test_files = glob.glob(os.path.join(serac_integration_test_dir, "*", "*.ats"))
+test_files = glob.glob(os.path.join(serac_integration_test_dir, "*/*", "*.ats"))
+#test_files = glob.glob(os.path.join(serac_integration_test_dir, "default", "*.ats"))
```
Comment (white238) [inline integration/test.ats]: Remove debug code

File: integration/thermal_conduction/dyn_imp_solve_restart/dyn_imp_solve_restart.ats [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# other Serac Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+tolerance_test(name="dyn_imp_solve_restart_serial",
+    input_file="{0}/tests/thermal_conduction/dyn_imp_solve_restart.lua".format(serac_input_files_dir))
+
+tolerance_test(name="dyn_imp_solve_restart_parallel",
+    input_file="{0}/tests/thermal_conduction/dyn_imp_solve_restart.lua".format(serac_input_files_dir),
```
Comment (white238) [inline integration/thermal_conduction/dyn_imp_solve_restart/dyn_imp_solve_restart.ats:12]: In a follow up PR we need to add a `restart_test` for these

Comment (white238) [reply integration/thermal_conduction/dyn_imp_solve_restart/dyn_imp_solve_restart.ats:12]: https://github.com/LLNL/serac/issues/655

File: integration/add_ats.sh [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+    cd "./${_type}"
+
+    # for each test in directory "t"
+    for t in $(ls --color=no) ; do
+
+        # remove .lua from file name
+        t=$(echo $t | awk -F "." '{print $1}')
+
+        if [ $t == "dyn_amgx_solve" ] ||
```
Comment (jamiebramwell) [inline integration/add_ats.sh:30]: Is there a reason these are being skipped?

File: integration/add_ats.sh [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+           [ $t == "static_amgx_solve" ] ||
+           [ $t == "static_reaction_exact" ] ; then
+            
+            echo "Skipping $t!"
+            
+            continue
+        fi
+
+        #echo "Writing ats file for ${_type}/${t}..."
```
Comment (jamiebramwell) [inline integration/add_ats.sh]: Dead code.

File: integration/add_ats.sh [hunk 6]
```diff
@@ -0,0 +1,68 @@
+#! /bin/bash
```
Comment (jamiebramwell) [inline integration/add_ats.sh:8]: Copyright needed.

File: integration/add_ats.sh (hunk 4 above)
Comment (jamiebramwell) [reply integration/add_ats.sh:30]: Nevermind, just saw the issues. AMGx only works on GPU machines. I will investigate the static reaction one.

Comment (chapman39) [reply integration/add_ats.sh:30]: the static reaction one times out when i run it, which is why I skip that one. https://github.com/LLNL/serac/issues/654

Comment (jamiebramwell) [reply integration/add_ats.sh:30]: Yeah, that shouldn't be timing out. I'll see what's up.

File: integration/test.ats [hunk 7]
```diff
@@ -78,36 +76,45 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
...
+            clas=checker_args,
+            label=name,
+            nosrun=True)
+    else:
+        # Rebaseline
+        if os.path.exists(test_summary_file):
+            print("Rebaselining {0}".format(name))
+            shutil.copyfile(test_summary_file, baseline_summary_file)
+        else:
+            print("Warning: tried to copy test file that doesn't exist:\n{0}".format(
```
Comment (chapman39) [inline integration/test.ats:108]: updated warning message so that the file directory is on a new line

File: integration/test.ats (hunk 2 above)
Comment (chapman39) [reply integration/test.ats]: this has been fixed https://github.com/LLNL/serac_tests/blob/0a820702e3e974730e69f38906aa3260727c12e4/integration/test.ats

**Review by white238 — APPROVED** [review]

**Review by jamiebramwell — APPROVED** [review]

**Pull request merged**